applications/RomApplication/python_scripts/rom_analysis.py

```diff
@@ -10,18 +10,36 @@
 
 from glob import glob
 from os import remove
+from pathlib import Path
 
-def CreateRomAnalysisInstance(cls, global_model, parameters):
+def CreateRomAnalysisInstance(cls, global_model, parameters, rom_training_parameters):
     class RomAnalysis(cls):
 
         def __init__(self,global_model, parameters):
+            # Enables loading the corresponding folder and file (defined on the rom manager).
+            self.rom_training_parameters = rom_training_parameters
             super().__init__(global_model, parameters)
 
         def _CreateSolver(self):
             """ Create the Solver (and create and import the ModelPart if it is not alread in the model) """
 
+            # Assign rom basis output folder and file name
+            if rom_training_parameters == None:
+                self.rom_basis_output_name = 'RomParameters' #Default
+                self.rom_basis_output_folder = 'rom_data' #Default
+                if self.project_parameters.Has("output_processes"):
```

Comment:
Maybe we should throw an error if there are no `output_processes`.

Reply:
No, because you could be using the rom_manager for this.